packages/aws-cdk/lib/api/bootstrap/bootstrap-template.json

```diff
@@ -0,0 +1,394 @@
+{
```

Comment:
Did we decide to add an optional resource name prefix (qualifier) or not?

Reply:
We decided not to do it in the bootstrap RFC.